metrotranscom/doorway
# feat: release lottery apps fe 2

This PR releases the following commits 

- feat: application side nav https://github.com/bloom-housing/bloom/pull/4237
- feat: feature flag and new endpoint https://github.com/bloom-housing/bloom/pull/4252
- fix: build resolution https://github.com/bloom-housing/bloom/pull/4278
- fix: clean up dto https://github.com/bloom-housing/bloom/pull/4281
- feat: lottery public list view https://github.com/bloom-housing/bloom/pull/4286

## How Can This Be Tested/Reviewed?

This PR can be tested by creating a public user and applying to listings that are open (with and without application due dates), closed (with and without lottery dates), and lottery run (triggered by publishing to public. Ensure the side nav, statues and buttons all show as expected.

UPDATE ON NOTE BELOW: Em pointed me to this [document](https://www.figma.com/design/Owt7RS8r1ESz3XhojupJVh/dwy-seeds?node-id=1-5166&node-type=CANVAS&m=dev) which detailed how to translate the tags on the left to the desired appearance on the right. The most recent commit handles that.

PLEASE NOTE: The variable overrides are creating contrast errors and I am discussing with design on how these should be addressed.

![Screenshot 2024-09-06 at 6 49 23 PM](https://github.com/user-attachments/assets/8c0b2f34-5354-49b4-a59a-c2cd322c4e30)
